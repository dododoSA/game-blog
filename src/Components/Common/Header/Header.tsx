import { SiteLogo } from 'Components/Common/Header/l1/SiteLogo';
import { HeaderMenu } from 'Components/Common/Header/l2/HeaderMenu';

const Header = () => (
  <header>
    <div>
      <SiteLogo />
    </div>
    <div>
      <HeaderMenu />
    </div>
  </header>
);

export { Header };
