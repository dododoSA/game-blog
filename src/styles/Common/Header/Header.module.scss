.logo_wrapper {
  text-align: center;
  margin: 3.0rem auto;
}