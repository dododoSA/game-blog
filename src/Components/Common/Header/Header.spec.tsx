import renderer from 'react-test-renderer';
import { Header } from 'Components/Common/Header/Header';

jest.mock('next/router', () => ({
  useRouter() {
    return {
      pathname: '/theory',
    };
  },
}));

test('Header', () => {
  const component = renderer.create(<Header />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
