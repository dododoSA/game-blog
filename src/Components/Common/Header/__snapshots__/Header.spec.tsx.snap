// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header 1`] = `
<header>
  <div
    className="logo_wrapper"
  >
    <div
      className="container"
    >
      <a
        className="logo"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Rのゲームプログラミング教室
      </a>
    </div>
  </div>
  <div>
    <nav
      className="nav_bar"
    >
      <ul>
        <li>
          <a
            className="menu_item current"
            href="/theory"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            理論編
          </a>
        </li>
        <li>
          <a
            className="menu_item"
            href="/practice"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            実践編
          </a>
        </li>
        <li>
          <a
            className="menu_item"
            href="/about"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            About
          </a>
        </li>
      </ul>
    </nav>
  </div>
</header>
`;
